certification/runtime/runner.go

```diff
@@ -6,10 +6,26 @@ import (
 	"github.com/komish/preflight/certification"
 	"github.com/komish/preflight/certification/errors"
 	"github.com/komish/preflight/certification/internal/policy"
-	"github.com/komish/preflight/version"
 )
 
-// TODO: Decide what of this file actually needs exporting
+// Register all policies
+var NameToPoliciesMap = map[string]certification.Policy{
+	policy.RunAsNonRootPolicy{}.Name():     policy.RunAsNonRootPolicy{},
+	policy.UnderLayerMaxPolicy{}.Name():    policy.UnderLayerMaxPolicy{},
+	policy.HasRequiredLabelPolicy{}.Name(): policy.HasRequiredLabelPolicy{},
+	policy.BasedOnUbiPolicy{}.Name():       policy.BasedOnUbiPolicy{},
+}
+
+func GetPoliciesByName() []string {
+	all := make([]string, len(NameToPoliciesMap))
+	i := 0
+
+	for k := range NameToPoliciesMap {
+		all[i] = k
+		i++
+	}
+	return all
+}
```

Comment:
This was previously the `AllPolicies` from the `certification` package, correct? I'm okay having moved it here for the moment, but let's leave it under the original name of `AllPolicies`. `GetPoliciesByName` implies that we're passing in a name to get some "thing", but that's not the case here.